DP-64: Add meta descriptions to each page in the operation guide. (#6376)
OpenNMS/opennms: docs/modules/operation/pages/deep-dive/events/sources/kafka.adoc

[[ga-events-sources-kafka]]

= Kafka Consumer (Events)

> :description: How to create events in OpenNMS Horizon/Meridian by routing them to a specific topic on Kafka: Kafka consumer and topic names.

Reviewer: ```suggestion
:description: How to ingest events into OpenNMS Horizon/Meridian by consuming a specified Kafka topic.
```
